data/fields/lane_markings.json

```
@@ -0,0 +1,5 @@
+{
+    "key": "lane:markings",
```

This key appears to be used [only once in OSM](https://taginfo.openstreetmap.org/keys/lane:markings). [`lane_markings`](https://wiki.openstreetmap.org/wiki/Key:lane_markings) is heavily used though:

```suggestion
    "key": "lane_markings",
```

---

Thanks for catching this!